cli/cmd.js

```diff
@@ -33,8 +34,12 @@ program.command("delete <type> <id>")
     .description("delete objects")
     .action(require("./delete.js"));
 
-program.command("update <type> <id> <file>")
+program.command("update <type> <id> [file]")
     .description("Update an object in the repository based on the JSON formatted data in a file")
     .action(require("./update.js"));
 
+program.command("create [file]")
+    .description("Create a new object of type <type> based on the JSON data in <file>")
```

Comment:
@andreas-marschke there's no `type` parameter; the type is simply based on whatever's in the JSON input.